csharp/ql/test/library-tests/definitions/definitions.expected

```diff
@@ -53,10 +53,10 @@
 | definitions.cs:114:28:114:28 | T | definitions.cs:114:17:114:17 | T | T |
 | definitions.cs:114:32:114:32 | A | definitions.cs:97:11:97:11 | A | T |
 | definitions.cs:117:27:117:27 | T | definitions.cs:117:18:117:18 | T | T |
-| definitions.cs:119:20:119:21 | I2<Object> | definitions.cs:117:15:117:19 | I2<> | T |
+| definitions.cs:119:20:119:29 | I2<Object> | definitions.cs:117:15:117:19 | I2<> | T |
 | definitions.cs:121:18:121:18 | A | definitions.cs:97:11:97:11 | A | T |
 | definitions.cs:121:21:121:22 | I1 | definitions.cs:112:15:112:16 | I1 | T |
-| definitions.cs:121:25:121:26 | I2<A> | definitions.cs:117:15:117:19 | I2<> | T |
+| definitions.cs:121:25:121:29 | I2<A> | definitions.cs:117:15:117:19 | I2<> | T |
```

Comment:
This change is problematic, as we will now have overlapping location spans for the character `A`: one for `I2<A>` (which jumps to `I2<>`) and one for `A` (which jumps to `A`), so not sure if the IDE/LGTM will work correctly when trying to jump to the definition for `A`. I agree that the right location for `I2<A>` is the whole span, so perhaps the fix instead is to adjust `hasLocationInfo` for constructed types to exclude type arguments, just like we [remove qualifiers from method calls](https://github.com/github/codeql/blob/main/csharp/ql/src/definitions.qll#L65).

Reply:
Thanks for noticing. I pushed a fix for this.